### ISSUE TITLE
Index name prefix problem in config.yml

### ISSUE THREAD

**Issue by lefakir** [algolia/AlgoliaSearchBundle#47]
I set up the bundle as mentionned in the documentation (Fill in your Algolia credentials) : 

```
# Algolia
algolia:
    catch_log_exceptions: true
    index_name_prefix: staging
```

And i get this error : 

```
  [Symfony\Component\Config\Definition\Exception\InvalidConfigurationException]
  Unrecognized option "index_name_prefix" under "algolia"
```

**Comment by maxiloc** [comment]
@lefakir What version of the package are you using ?

**Comment by lefakir** [comment]
Hi,
In composer.lock it is **1.0.13** and in composer.json it is set to `"algolia/algolia-search-bundle": "^1.0"` 

In parameters.yml i also tried to set : 
```
algolia.index_name_prefix: local
```
There is no error but the index feeded in the algolia dashboard is always _MyEntityName_dev_.
I expect this to be _MyEntityName_local_ our _local_MyEntityName_, right ?

Hope it helps

**Comment by hmartin** [comment]
@lefakir fix in 2.0.0
